### PR TITLE
LEARNER-4931 Test - Android - Automate Main Dashboard Screen

### PR DESCRIPTION
**Pull request by nedo**
Following features/stories have been implemented,

* Verify that after successful login Main Dashboard screen is loaded
* Verify that Main Dashboard is loaded successfully and have following contents,
** Logged in user"s avatar
** Screen Title
** Account Icon
** 3 tab options below
** Courses
** Discovery
** Courses tab will be selected by default
* Verify all elements have default value
* Verify on tapping Courses will load Courses contents in its tab
* Verify on tapping Discovery will load Discovery contents in its tab
* Verify tapping user"s avatar will load User Profile screen
* Verify tapping back/cancel icon from User Profile screen should get back to Main Dashboard screen
* Verify tapping Account Icon will load Account Screen
* Verify tapping back/cancel icon from Account Screen should get back to Main Dashboard screen
* Verify that after re-login enrolled Courses will be available

### CODE REVIEW

**Comment by nedo** [comment]
@raeeschachar PR is ready for review
cc @staubina

File: android/tests/test_android_main_dashborad.py [hunk 1]
```diff
@@ -25,23 +25,62 @@ def test_start_main_dashboard_smoke(self, login, set_capabilities, setup_logging
...
-        Scenarios:
-                Verify following contents are visible on screen
-                     Screen Title, Menu Drawer, Account Menu option and Log out user
-                Verify all screen contents have their default values
-                Verify that user can log out successfully, and back on Login screen
+         Scenarios:
+                Verify following contents are visible on screen, 
+                    Logged in user's avatar, Screen Title, Account Icon
+                    Courses Tab, Discovery Tab
+                Verify that Courses tab will be selected by default
```
Comment (raeeschachar) [inline android/tests/test_android_main_dashborad.py]: You need to add encoding to this file i.e `# coding=utf-8`

File: android/tests/test_android_main_dashborad.py [hunk 2]
```diff
@@ -25,23 +25,62 @@ def test_start_main_dashboard_smoke(self, login, set_capabilities, setup_logging
...
+    def test_logout_smoke(self, set_capabilities, setup_logging):
+        """
+        Scenario:
+                Verify that user can log out successfully, and back on Login screen
+        """
+
+        android_main_dashboard_page = AndroidMainDashboard(set_capabilities, setup_logging)
+        global_contents = Globals(setup_logging)
+
+        # assert android_main_dashboard_page.get_account_options()[3].text == strings.ACCOUNT_LOGOUT
```
Comment (raeeschachar) [inline android/tests/test_android_main_dashborad.py]: Was this comment left intentionally?

File: android/tests/test_android_main_dashborad.py (hunk 1 above)
Comment (nedo) [reply android/tests/test_android_main_dashborad.py]: added encoding to all files where ever it was recommended

File: android/tests/test_android_main_dashborad.py (hunk 2 above)
Comment (nedo) [reply android/tests/test_android_main_dashborad.py]: removed this point as it is no more needed.

**Comment by nedo** [comment]
@staubina PR is ready for review

**Pull request merged**